ItemsListApp/src/ItemsListApp.Contracts/Services/IIdGeneratorService.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace ItemsListApp.Contracts.Services
+{
+    public interface IIdGeneratorService
```

Comment:
The contract actually serves you `ids` rather than `generators`. Name should follow the concept...